ssl/statem/statem_clnt.c

```diff
@@ -3456,6 +3456,11 @@ MSG_PROCESS_RETURN tls_process_hello_req(SSL *s, PACKET *pkt)
         return MSG_PROCESS_ERROR;
     }
 
+    if ((SSL_get_options(s) & SSL_OP_NO_RENEGOTIATION)) {
```

Comment:
In the s_server.c, we kinda need to use SSL_get_options(), within the library, not so much.